accessibility-checker-extension/src/ts/devtools/ReportRow.tsx

```diff
@@ -70,6 +76,28 @@ export default class ReportRow extends React.Component<IReportRowProps, IReportR
         }
     }
 
+    learnMoreClickHandler = (e:any, item:IReportItem)=>{
```

Comment:
This is a class. Please use class function syntax:
```learnMoreClickHandler(e: any, item: IReportItem) {```
Ideally, find the type of e, rather than using `any`

Reply:
changed class function syntax